docs/podtemplates.md

```diff
@@ -95,6 +95,40 @@ Pod templates support fields listed in the table below.
 	</tbody>
 </table>
 
+## Use <code>imagePullSecrets</code> to lookup entrypoint
```

Comment:
nit: In Markdown, we can use the ` character to enable code mode, without the more verbose/distracting HTML (here and below).


```suggestion
## Use `imagePullSecrets` to lookup entrypoint
```

Reply:
Thank you!